client/whiteboard/state/useBoardState.ts

```diff
@@ -89,6 +90,14 @@ const reducer = (state: BoardState, action: BoardAction): BoardState => {
           };
       }
 
+      case "move":
+        if (state.activeUser.name != action.payload.userName){
```

Comment:
user names are not enforced to be unique at this time, but user ids are - let's check for that here.